### DISCUSSION TITLE
About license for diagrams generated with drawio-desktop

### DISCUSSION THREAD

**Discussion by meowfishorg** [jgraph/drawio-desktop#1302] [General]
The license is Apache 2.0. 

If I want to use drawio-desktop to draw diagrams and use those diagrams in my document (and selling those documents), is this permission also granted by default? Just want to be sure that this action is legit and free of charge.

**Comment by davidjgraph** [comment]
Yes, we make no copyright claim on the resulting diagrams as a whole. Frankly, even if we tried to, there's no way that'd work in any modern legal system.
